tests/gis_tests/geoapp/models.py

```diff
@@ -95,3 +95,7 @@ class ManyPointModel(NamedModel):
     point1 = models.PointField()
     point2 = models.PointField()
     point3 = models.PointField(srid=3857)
+
+
+class ThreeDimensionalFeature(NamedModel):
+    geom = models.GeometryField(dim=3)
```

Comment:
I added `required_db_features = {"supports_3d_storage"}`, check out fed257ddff449e1bf1ce06d9a7fddb148290a6ac.

Reply:
All tests pass on CockroachDB. Since the commit you mentioned, CockroachDB added support for 3D storage so the feature flag isn't used there any longer.

Reply:
Many thanks for checking :+1: